python/lsst/ap/association/afwUtils.py

```diff
@@ -59,54 +59,453 @@ def make_minimal_dia_object_schema(filter_names=None):
+                    doc='Standard error of iPSFluxMean.')
+    schema.addField('iPSFluxSigma', type='D',
+                    doc='Standard deviation of the distribution of iPSFlux.')
+    schema.addField('iPSFluxChi2', type='D',
+                    doc='Chi^2 statistic for the scatter of iPSFlux around iPSFluxMean.')
+    schema.addField('iPSFluxNdata', type='I',
+                    doc='The number of data points used to compute iPSFluxChi2.')
+    schema.addField('iFPFluxMean', type='D',
+                    doc='Weighted mean forced photometry flux for i filter.')
+    schema.addField('iFPFluxMeanErr', type='D',
+                    doc='Standard error of iFPFluxMean.')
+    schema.addField('iFPFluxSigma', type='D',
+                    doc='Standard deviation of the distribution of iFPFlux.')
+    schema.addField('zPSFluxMean', type='D',
+                    doc='Weighted mean point-source model magnitude for z filter.')
+    schema.addField('zPSFluxMeanErr', type='D',
+                    doc='Standard error of zPSFluxMean.')
+    schema.addField('zPSFluxSigma', type='D',
+                    doc='Standard deviation of the distribution of zPSFlux.')
+    schema.addField('zPSFluxChi2', type='D',
+                    doc='Chi^2 statistic for the scatter of zPSFlux around zPSFluxMean.')
+    schema.addField('zPSFluxNdata', type='I',
+                    doc='The number of data points used to compute zPSFluxChi2.')
+    schema.addField('zFPFluxMean', type='D',
+                    doc='Weighted mean forced photometry flux for z filter.')
+    schema.addField('zFPFluxMeanErr', type='D',
+                    doc='Standard error of zFPFluxMean.')
+    schema.addField('zFPFluxSigma', type='D',
+                    doc='Standard deviation of the distribution of zFPFlux.')
+    schema.addField('yPSFluxMean', type='D',
+                    doc='Weighted mean point-source model magnitude for y filter.')
+    schema.addField('yPSFluxMeanErr', type='D',
+                    doc='Standard error of yPSFluxMean.')
+    schema.addField('yPSFluxSigma', type='D',
+                    doc='Standard deviation of the distribution of yPSFlux.')
+    schema.addField('yPSFluxChi2', type='D',
+                    doc='Chi^2 statistic for the scatter of yPSFlux around yPSFluxMean.')
+    schema.addField('yPSFluxNdata', type='I',
+                    doc='The number of data points used to compute yPSFluxChi2.')
+    schema.addField('yFPFluxMean', type='D',
+                    doc='Weighted mean forced photometry flux for y filter.')
+    schema.addField('yFPFluxMeanErr', type='D',
+                    doc='Standard error of yFPFluxMean.')
+    schema.addField('yFPFluxSigma', type='D',
+                    doc='Standard deviation of the distribution of yFPFlux.')
+    # Mapping of arrays and BLOBs is not currently supported by the PPDB so we
+    # these columns out.
+    """
+    schema.addField('uLcPeriodic', type='ArrayF',
```

Comment:
Same. I'll change these to normal comments.

Reply:
It was a quick and dirty solution to comment out the block because Ppdb doesn't currently support array types or BLOBs. Forgot to change it for the PR.